src/desktop/apps/fair/components/browse/router.coffee

```diff
@@ -47,7 +47,7 @@ module.exports = class BrowseRouter extends Backbone.Router
     mediator.trigger 'open:auth',
       mode: 'register'
       copy: "Sign up to receive updates about #{@fair.nameSansYear()}"
-      redirectTo: "#{@fair.href()}/capture/#{action}"
+      destination: "#{@fair.href()}/capture/#{action}"
```

Comment:
This is a special view: https://www.artsy.net/arco-madrid-2018/sign_up (which I believe are linked to in some emails/other places?), where the modal is open for you, and in this case, we want to go to onboarding (so the `redirectTo` -> `destination` change), but we want to keep this special server-side destination route, as it does a bunch of work under the hood: https://github.com/artsy/force/blob/92f13640f8d21c6050d2e5a28fb98df8dcebe500/src/desktop/apps/fair/routes.coffee#L114-L117